commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -535,6 +545,7 @@ def get_product_by_program_id(self, program_id: str) -> Optional[ProductVariant]
 
         return results[0] if results else None
 
+    @conditional_retry
     def get_product_variant_by_course_run(self, cr_id: str) -> Optional[ProductVariant]:
```

Comment:
Can we confirm the places where this function is being used and if there are any ripple effects of putting conditional_retry on it?